src/MonitorProfiler/Environment/EnvironmentHelper.h

```diff
@@ -14,9 +14,17 @@
 class EnvironmentHelper final
 {
 private:
+    static constexpr LPCWSTR s_wszDebugLoggerLevelEnvVar = _T("DotnetMonitorProfiler_DebugLogger_Level");
```

Comment:
Should this be `DotnetMonitorProfiler_DebugLogger__Level` (two underscores before Level), or is this correct as-is?

Reply:
These variables are not used in configuration, so the double underscore is not necessary.